newIDE/app/src/MainFrame/index.js

```diff
@@ -1583,8 +1574,11 @@ class MainFrame extends React.Component<Props, State> {
     });
   };
 
-  _setLayoutForPreview = (name:string) => {
-    
+  _setLayoutForPreview = (name?: string) => {
+    if (!name) {
+      const editorTab = getCurrentTab(this.state.editorTabs);
+      name = editorTab.name;
+    }
```

Comment:
Explaination are here
https://github.com/4ian/GDevelop/pull/1429#discussion_r382560006

Reply:
This is now deleted. See last commit.
Same behavior as in explaination but works differently.